### PR TITLE
linking between NB 01-08 & from manuscript to NB 01-08

### PR DESCRIPTION
**Pull request by thempel**
#170 

Is this enough? We should probably go into more details especially about the sections mentioned in the above issue. I suppose @psolsson can do a better summary of the experimental observable notebook.

### CODE REVIEW

File: manuscript/manuscript.tex [hunk 1]
```diff
@@ -398,6 +399,8 @@ \subsection{Dimensionality reduction}
 Discrete jumps between the minima can be observed by visualizing the transformation of the first trajectory into these ICs (Fig.~\ref{fig:io-to-tica}d).
 We thus assume that our TICA-transformed backbone torsion features describe one or more metastable processes.
 
+We demonstrate how to apply TICA, suggest how to interpret the projected coordinates and compare the results to other dimension reduction techniques in Notebook (02).
```
Comment (brookehus) [inline manuscript/manuscript.tex]: Have we in general omitted oxford commas from the manuscript? As long as it's consistent, all good.

File: manuscript/manuscript.tex [hunk 2]
```diff
@@ -414,7 +417,8 @@ \subsection{Discretization}
 which can greatly facilitate the decomposition of our system into the discrete Markovian states necessary for MSM estimation.
 Here, we use the $k$-means algorithm to segment the four dimensional TICA space into $k=75$ cluster centers.
 The number of cluster centers has been chosen to optimize the VAMP-2 score in a manner identical to how the feature selection was carried out above,
-which is shown in the showcase notebook (00).
+which is shown in the showcase Notebook (00).
+A detailed comparison between different clustering techniques is provided in Notebook (02).
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "among different clustering techniques" if more than 2 are compared

File: manuscript/manuscript.tex [hunk 3]
```diff
@@ -560,6 +568,8 @@ \subsection{Connecting the MSM with experimental data}
 We see that the predicted relaxation signal has a much larger amplitude for the nonequilibrium initialization,
 making it more likely to be experimentally measurable.
 
+Besides a detailed demonstration of the above, Notebook (06) demonstrates how to compute J-couplings and dynamic fingerprints from MSMs.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "In addition to a detailed demonstration" ?

File: manuscript/manuscript.tex [hunk 4]
```diff
@@ -587,6 +597,7 @@ \subsection{Modeling large systems}
 we explain how to deal with those in the tutorials (Notebook 00 and 02).
 
 More details on how to model complex systems with the techniques presented here are described, e.g., by~\cite{plattner_protein_2015,plattner_complete_2017}.
+We further demonstrate the symptoms of difficult data situations and how to deal with them in Notebook (08).
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "difficult data situations" sounds awkward to me.

Maybe:

"We further examine some symptoms that may indicate problematic or difficult datasets, and demonstrate how to deal with them ..." ?

File: notebooks/02-dimension-reduction-and-discretization.ipynb [hunk 5]
```diff
@@ -387,7 +388,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Following the previous example, we perform a $k$-means ($100$ centers, stride of $5$) and a regspace clustering ($0.3$ radians center distance) on the full two-dimensional data set and visualize the obtained centers:"
+    "Following the previous example, we perform a $k$-means ($100$ centers, stride of $5$) and a regspace clustering ($0.3$ radians center distance) on the full two-dimensional data set and visualize the obtained centers. In [Notebook 03 ➜ 📓](03-msm-estimation-and-validation.ipynb), we show the effect of different numbers of cluster centers."
```
Comment (brookehus) [inline notebooks/02-dimension-reduction-and-discretization.ipynb]: The effect of different numbers of cluster centers on what?

File: notebooks/03-msm-estimation-and-validation.ipynb [hunk 6]
```diff
@@ -240,7 +240,8 @@
     "\n",
     "Please note though that this ITS convergence analysis is based on the assumption that $200$ $k$-means centers are sufficient to discretize the dynamics.\n",
     "In order to study the influence of the clustering on the ITS convergence,\n",
-    "we repeat the clustering and ITS convergence analysis for various number of cluster centers:"
+    "we repeat the clustering and ITS convergence analysis for various number of cluster centers.\n",
+    "For the sake of simplicity, we will restrict ourselves to the $k$-means algorithm; different techniques are presented in [Notebook 02 ➜ 📓](02-dimension-reduction-and-discretization.ipynb)."
```
Comment (brookehus) [inline notebooks/03-msm-estimation-and-validation.ipynb]: "different techniques" > "alternative clustering methods"

File: notebooks/08-common-problems.ipynb [hunk 7]
```diff
@@ -781,7 +781,7 @@
     "the TICA lag time was deliberately chosen way too high.\n",
     "That's easy to fix.\n",
     "\n",
-    "Let's now have a look at how the metastable trajectories should look like for a decent model such as the one estimated in the previous notebooks.\n",
+    "Let's now have a look at how the metastable trajectories should look like for a decent model such as the one estimated in [Notebook 05 ➜ 📓](05-pcca-tpt.ipynb).\n",
```
Comment (brookehus) [inline notebooks/08-common-problems.ipynb]: "how the metastable trajectories should look for a decent model"

File: manuscript/manuscript.tex (hunk 1 above)
Comment (thempel) [reply manuscript/manuscript.tex]:
```suggestion
We demonstrate how to apply TICA, suggest how to interpret the projected coordinates, and compare the results to other dimension reduction techniques in Notebook (02).
```

File: manuscript/manuscript.tex (hunk 3 above)
Comment (thempel) [reply manuscript/manuscript.tex]:
```suggestion
In addition to a detailed demonstration of the above, Notebook (06) demonstrates how to compute J-couplings and dynamic fingerprints from MSMs.
```

File: manuscript/manuscript.tex (hunk 4 above)
Comment (thempel) [reply manuscript/manuscript.tex]:
```suggestion
We further examine some symptoms that may indicate problematic or difficult datasets, and demonstrate how to deal with them in Notebook (08).
```

File: notebooks/02-dimension-reduction-and-discretization.ipynb (hunk 5 above)
Comment (thempel) [reply notebooks/02-dimension-reduction-and-discretization.ipynb]:
```suggestion
    "Following the previous example, we perform a $k$-means ($100$ centers, stride of $5$) and a regspace clustering ($0.3$ radians center distance) on the full two-dimensional data set and visualize the obtained centers. In [Notebook 03 ➜ 📓](03-msm-estimation-and-validation.ipynb), we show the effect of different numbers of cluster centers on MSM estimation."
```

File: notebooks/03-msm-estimation-and-validation.ipynb (hunk 6 above)
Comment (thempel) [reply notebooks/03-msm-estimation-and-validation.ipynb]:
```suggestion
    "For the sake of simplicity, we will restrict ourselves to the $k$-means algorithm; alternative clustering methods are presented in [Notebook 02 ➜ 📓](02-dimension-reduction-and-discretization.ipynb)."
```

File: notebooks/08-common-problems.ipynb (hunk 7 above)
Comment (thempel) [reply notebooks/08-common-problems.ipynb]:
```suggestion
    "Let's now have a look at how the metastable trajectories should look for a decent model such as the one estimated in [Notebook 05 ➜ 📓](05-pcca-tpt.ipynb).\n",
```

File: manuscript/manuscript.tex (hunk 2 above)
Comment (thempel) [reply manuscript/manuscript.tex]: only 2 are compared

**Comment by thempel** [comment]
Sorry for the spam, I was testing a new github feature.

**Comment by brookehus** [comment]
-1 for new github feature

**Pull request merged**